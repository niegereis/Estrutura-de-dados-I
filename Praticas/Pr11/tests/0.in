3
5 1 3 5 2 1
pedro 2014381 29 90.0
guilherme 2213025 25 70.0
daiane 2021235 30 50.0