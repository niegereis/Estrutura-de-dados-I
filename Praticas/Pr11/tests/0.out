[2021235] daiane - 30 (50.0)
[2014381] pedro - 29 (90.0)
[2213025] guilherme - 25 (70.0)